src/components/kubectl/port-forward.ts

```diff
@@ -203,18 +225,38 @@ async function findPortForwardablePods (): Promise<PortForwardFindPodsResult> {
 }
 
 /**
- * Invokes kubectl port-forward.
- * @param podName The pod name.
- * @param portMapping The PortMapping object. Provides the local and target ports.
- * @param namespace  The namespace to use to find the pod in.
- * @returns The locally bound port that was used.
+ * Invokes kubectl port-forward
+ * @param podName The pod name
+ * @param portMapping The PortMapping objects. Each object contains the requested local and target ports
```

Comment:
@itowlson there are two places where it can get assigned:

1. After port prompt, after validation, if the user has provided both the local and target port
2. If the user has *not* provided the local port at the port prompt step, we pick one for them in the `portForwardToPod` function.